### PR TITLE
[write_standby] update write_standby.py script

### PR DESCRIPTION
**Pull request by yxieca**
#### Why I did it
The initial value has to be present for the state machines to work. In active-standby dual-tor scenario, or any hardware mux scenario, the value will be updtaed eventually with a delay.

However, in active-active dual-tor scenario, there is no other mechanism to initialize the value and get state machines started.
So this script will have to write something at start up time.

For active-active dualtor, 'active' is a more preferred initial value, the state machine will switch the state to standby soon if
link prober found link not in good state.

#### How I did it
Update the script to always provide initial values.

#### How to verify it
Tested on active-active dual-tor testbed.

Signed-off-by: Ying Xie <ying.xie@microsoft.com>

### CODE REVIEW

**Review by zjswhhh — APPROVED** [review]
LGTM.

File: files/scripts/write_standby.py [hunk 1]
```diff
@@ -96,14 +96,21 @@ def is_warmrestart(self):
...
         all_intfs = self.config_db.get_table('MUX_CABLE')
-        auto_intfs = [intf for intf, status in all_intfs.items()
-                                if status['state'].lower() == 'auto']
-        return auto_intfs
+        for intf, status in all_intfs.items():
+            state = status['state'].lower()
+            if state == 'standby':
+                intf_modes[intf] = 'standby'
+            elif state in ['active', 'auto', 'manual']:
+                intf_modes[intf] = 'active'
```
Comment (prsunny) [inline files/scripts/write_standby.py:112]: how does this behave for y-cable dual-tor?

Comment (yxieca) [reply files/scripts/write_standby.py]: updated. thanks.

File: files/scripts/write_standby.py [hunk 2]
```diff
@@ -144,19 +157,23 @@ def apply_mux_config(self):
...
             logger.log_error("Timed out waiting for tunnel {}, mux state will not be written".format(self.tunnel_name))
 
 
 if __name__ == '__main__':
-    mux_writer = MuxStateWriter()
+    parser = argparse.ArgumentParser(description='Write initial mux state')
+    parser.add_argument('-s', '--state',
+                        help='state: intial state for "auto" and/or "manual" config, default "standby"',
+                        type=str, required=False, default='standby')
+    args = parser.parse_args()
```
Comment (prsunny) [inline files/scripts/write_standby.py:177]: i think we've few places where write_standby script is used/planned to use.  So just want to confirm that we tested the case if no param given, it is default to 'standby'.

**Review by prsunny — APPROVED** [review]
lgtm, @theasianpianist please review/sign-off.

File: files/scripts/write_standby.py (hunk 2 above)
Comment (yxieca) [reply files/scripts/write_standby.py:177]: yes. I tested that.

**Review by theasianpianist — APPROVED** [review]
lgtm! Wondering if with these changes it would be appropriate to rename this script to something like `write_mux_state`?

**Pull request merged**